src/modules/dashboard/components/confirmation-dialog.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <v-dialog v-model="isDialogVisible">
+    <div class="container">
+      <section class="container--header">
+        <h2>{{ title }}</h2>
+      </section>
+      <section class="container--content">
+        <p>{{ message }}</p>
+      </section>
+      <section class="container--footer">
```

Comment:
container__header, __content, __footer. to nie są modyfikatory lecz elementy komponentu, więc podwójna podłoga byłaby tutaj lepsza. 

Poza tym, zastanawiam się nad lepszą semantyką, co myślisz o tym aby cały dialog (od samej góry) wyglądał tak:
section
  header
  main
  footer

:)

Reply:
Hmm, w sumie racja, mój błąd. Pamiętam, że czytałem kiedyś o tym kiedy nie warto zagnieżdżać/dodawać kolejnych headerów na stronie, ale w sumie tutaj jedyne co tracimy to banner landmark, którego nie potrzebuję.

Reply:
`Yes, but with a catch. The W3 documents state that the tags represent the header and footer areas of their nearest ancestor section.`